site/docs/configuration.md

```diff
@@ -40,6 +40,9 @@ Iceberg tables support table properties to configure table behavior, like the de
 | write.parquet.dict-size-bytes      | 2097152 (2 MB)     | Parquet dictionary page size                       |
 | write.parquet.compression-codec    | gzip               | Parquet compression codec                          |
 | write.parquet.compression-level    | null               | Parquet compression level                          |
+| write.parquet.bloom-filter-enabled | false | Whether to enable writing bloom filter; If it is true, the bloom filter will be enable for all columns; If it is false, it will be disabled for all columns; It is also possible to enable it for some columns by specifying the column name within the property followed by #; For example, setting both `write.parquet.bloom-filter-enabled=true` and `write.parquet.bloom-filter-enabled#some_column=false` will enable bloom filter for all columns except `some_column` |
```

Comment:
Minor suggestions. How about this?
```
To enable or disable bloom filter for all columns by default. Partial enabling and disabling are possible by specifying the column name. For example, ...
```

Reply:
BTW, how does a user specify multiple columns? like this or something else?
```
write.parquet.bloom-filter-enabled#column1=false
write.parquet.bloom-filter-enabled#column2=false
```

Reply:
Yes, the format references properties defined in [parquet-hadoop](https://github.com/apache/parquet-mr/tree/master/parquet-hadoop), like `parquet.bloom.filter.enabled` and `parquet.bloom.filter.expected.ndv`. 
However, according to Ryan's [comment](https://github.com/apache/iceberg/pull/2582#discussion_r650448248), I will change the property pattern to `write.parquet.bloom-filter.col1.enabled`.

Reply:
OK, it requires changes for the class `ColumnConfigParser` since the "#" isn't used.